src/test/java/org/reaktivity/nukleus/maven/plugin/internal/generated/IntegerFixedArraysFWTest.java

```diff
@@ -0,0 +1,352 @@
+/**
+ * Copyright 2016-2017 The Reaktivity Project
+ *
+ * The Reaktivity Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package org.reaktivity.nukleus.maven.plugin.internal.generated;
+
+import static java.nio.ByteBuffer.allocateDirect;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+import java.util.PrimitiveIterator;
+import java.util.PrimitiveIterator.OfInt;
+import java.util.PrimitiveIterator.OfLong;
+
+import org.agrona.MutableDirectBuffer;
+import org.agrona.concurrent.UnsafeBuffer;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.reaktivity.reaktor.internal.test.types.inner.IntegerFixedArraysFW;
+
+public class IntegerFixedArraysFWTest
+{
+    public class IntIterator implements OfInt
+    {
+        private final int[] values;
+        private int index = 0;
+
+        public IntIterator(int... values)
+        {
+            this.values = values;
+        }
+
+        @Override
+        public boolean hasNext()
+        {
+            return index < values.length;
+        }
+
+        @Override
+        public int nextInt()
+        {
+            return values[index++];
+        }
+    }
+
+    public class LongIterator implements OfLong
+    {
+        private final long[] values;
+        private int index = 0;
+
+        public LongIterator(long... values)
+        {
+            this.values = values;
+        }
+
+        @Override
+        public boolean hasNext()
+        {
+            return index < values.length;
+        }
+
+        @Override
+        public long nextLong()
+        {
+            return values[index++];
+        }
+    }
```

Comment:
Use `LongStream.of(...).iterator()` instead.

Reply:
done